plugin.py

```diff
@@ -1,14 +1,24 @@
 from LSP.plugin import AbstractPlugin
 from LSP.plugin import register_plugin
 from LSP.plugin import Request
+from LSP.plugin import Session
 from LSP.plugin import unregister_plugin
+from LSP.plugin.core.protocol import Range
 from LSP.plugin.core.registry import LspTextCommand
-from LSP.plugin.core.typing import Optional, Dict
+from LSP.plugin.core.types import debounced
+from LSP.plugin.core.types import FEATURES_TIMEOUT
+from LSP.plugin.core.typing import Optional, Dict, TypedDict, List, Tuple
+from LSP.plugin.core.views import point_to_offset
+from LSP.plugin.core.views import text_document_identifier
```

Comment:
I cannot guarantee stability with respect to these imports. Ideally only import from `LSP.plugin`. But I'll keep it in mind anyway :)

Reply:
Ideally some of these would be there but I didn't want to re-invent them. I might refactor these out later to remake them myself.